test/commands/runtime/action/list.test.js

```diff
@@ -98,8 +114,8 @@ describe('instance methods', () => {
       command.argv = ['--skip', '3']
       return command.run()
         .then(() => {
-          expect(cmd).toHaveBeenCalledWith({ skip: 3 })
-          expect(stdout.output).toMatch('actions                                           ')
+          expect(cmd).toHaveBeenCalledWith({ skip: 3, useragent: expect.any(String) })
```

Comment:
I wouldn't expect to test for `useragent` in this `runtime:list --skip` test? Maybe this should be in a separate test for `useragent` being set in a call. Perhaps this should be `expect.objectContaining({skip: 3})` instead